gbols/fast-food
# #160645401 Endpoint to sign up and sign out.

**What does this PR do?**
It enables a user to sign up for an account and sign out of the account.

**Description of Task to be completed?**
- create endpoint to log in a user.
- write test for logging in a user.

**How should this be manually tested?**
-pull from this branch `ft-endpoint-for-signup-signout-160645401
` into  a new branch.
-run npm install.
-the endpoint should look like this: `http://localhost:3000/api/v1/auth/signup` and `http://localhost:3000/api/v1/auth/signout`
-run npm test.

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
 #160645401 a user should be able to create an account to sign in to / sign out from the app.
Screenshots (if appropriate) 
![screenshot from 2018-09-23 21-21-50](https://user-images.githubusercontent.com/27724644/45932703-5a7e2f80-bf78-11e8-92b1-a90a86c84af4.png)
![screenshot from 2018-09-23 21-21-18](https://user-images.githubusercontent.com/27724644/45932704-5ce08980-bf78-11e8-836f-ce426ae764a5.png)
